EMIP/1001-2000/EMIP01950.xml

```diff
@@ -57,6 +64,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <handDesc><handNote xml:id="h1" script="Ethiopic"><seg type="rubrication">
+                    <foreign xml:lang="gez">ማርያም፡</foreign>
+                    <foreign xml:lang="gez">ሚካኤል፡</foreign>
+                    <foreign xml:lang="gez">ገብርኤል፡</foreign>
+                    <foreign xml:lang="gez">መስቀል፡</foreign>
+                    <foreign xml:lang="gez">ስላሤ፡</foreign>
+
+                  </seg>
+                </handNote></handDesc>
+                  <decoDesc>
+                    <decoNote type="frame" xml:id="d01">
```

Comment:
Sorry if I did not notice this before. I am not sure about the reason for which here a 2 digit number is used, so d01 instead of simply d1